src/cli/__tests__/__snapshots__/cli.test.ts.snap

```diff
@@ -0,0 +1,55 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`collect/visualize integration creates graph based on integration data 1`] = `
+"<!DOCTYPE html>
+    <html lang=\\"en\\">
+    <head>
+        <meta charset=\\"UTF-8\\">
+        <meta name=\\"viewport\\" content=\\"width=device-width, initial-scale=1.0\\">
+        <title>Integration SDK Graph</title>
+    </head>
+    <body style=\\"height:100vh;width:100%\\">
+        <div id=\\"integration-graph\\" style=\\"height:100vh;width:100%\\"></div>
+        <script type=\\"text/javascript\\" src=\\"https://unpkg.com/vis-network/standalone/umd/vis-network.min.js\\"></script>
+        <script type=\\"text/javascript\\">
+            var nodes = new vis.DataSet([{\\"id\\":\\"account:1234\\",\\"label\\":\\"Account 1234\\"},{\\"id\\":\\"user:12345\\",\\"label\\":\\"User 12345\\"}]);
+            var edges = new vis.DataSet([{\\"from\\":\\"account:1234\\",\\"to\\":\\"user:12345\\",\\"label\\":\\"HAS\\"}]);
+            // create a network
+            var container = document.getElementById(\\"integration-graph\\");
+            var data = {
+              nodes: nodes,
+              edges: edges
+            };
+            var options = {\\"edges\\":{\\"arrows\\":{\\"to\\":{\\"enabled\\":true}}}};
+            var network = new vis.Network(container, data, options);
+          </script>
+    </body>
+    </html>"
+`;
```

Comment:
I generally dislike snapshot testing (unless using actual image snapshots for component testing), but this is probably one of the few cases where I think I'm cool with it since it can actually help catch bad changes.

Reply:
Oh related to this. Can you add the [--ci](https://jestjs.io/docs/en/cli.html#--ci) flag to the `test:ci` script? It will help ensure that unwanted tweaks to the snapshot are caught.